src/aics-image-viewer/components/ChannelsWidgetRow/index.tsx

```diff
@@ -64,24 +61,24 @@ export default class ChannelsWidgetRow extends React.Component<ChannelsWidgetRow
     if (!target.checked && !isosurfaceChecked) {
       this.setState({ controlsOpen: false });
     }
-    changeOneChannelSetting(channelName, index, VOLUME_ENABLED, target.checked);
+    changeOneChannelSetting(channelName, index, "volumeEnabled", target.checked);
   }
 
   isosurfaceCheckHandler({ target }: { target: { checked: boolean } }) {
     const { channelName, index, changeOneChannelSetting, volumeChecked } = this.props;
     if (!target.checked && !volumeChecked) {
       this.setState({ controlsOpen: false });
     }
-    changeOneChannelSetting(channelName, index, ISO_SURFACE_ENABLED, target.checked);
+    changeOneChannelSetting(channelName, index, "isosurfaceEnabled", target.checked);
   }
 
-  createChannelSettingHandler = (settingKey: string) => (newValue: any) => {
+  createChannelSettingHandler = (settingKey: ChannelStateKey) => (newValue: any) => {
     const { channelName, index, changeOneChannelSetting } = this.props;
     changeOneChannelSetting(channelName, index, settingKey, newValue);
   };
 
-  onIsovalueChange = this.createChannelSettingHandler(ISO_VALUE);
-  onOpacityChangeUnwrapped = this.createChannelSettingHandler(OPACITY);
+  onIsovalueChange = this.createChannelSettingHandler("isovalue");
+  onOpacityChangeUnwrapped = this.createChannelSettingHandler("opacity");
```

Comment:
Are these strings `ChannelStateKey`s ?  it would be cool if there were a typescripty way to enforce that.  

If not, then I think passing these as string literals inline is worse than before, because suddenly the code is more error prone.  If we follow this up quickly with resolving that in another PR (by using an enum or reinstating symbolic constants somehow) then can let it be for now.

Reply:
As written, it is a type error to pass any string literal that isn't a `keyof ChannelState`. This isn't as great for refactoring, as you can't hit F2 on the property and have it change this string literal, but it'll still catch and flag the inconsistency when you do.